src/content/operations/troubleshooting/_index.en.md

```diff
@@ -15,6 +15,40 @@ Basic familiarity with the Submariner components and architecture will be helpfu
 
 The guide has been broken into different sections for easy navigation.
 
+## Automated Troubleshooting
+
+Use the `subctl` utility to automate troubleshooting and collecting debugging information.
+
+Install subctl:
+
+```shell
+curl -Ls https://get.submariner.io | VERSION=<your Submariner version> bash
+```
+
+Make sure `KUBECONFIG` is set to point at your connected clusters:
+
+```shell
+export KUBECONFIG=<paths>
+```
+
+Get an idea of what's failing:
+
+```shell
+subctl diagnose all
```

Comment:
Thanks for the PR @dfarrell07. Can you also include the following command.
`subctl diagnose firewall inter-cluster`